atomic_reactor/inner.py

```diff
@@ -337,6 +341,7 @@ def build_docker_image(self):
             start_time = datetime.datetime.now()
             self.plugins_timestamps['dockerbuild'] = start_time.isoformat()
 
+            signal.signal(signal.SIGTERM, self.throw_cancelled_build_exception)
```

Comment:
Agreed, moving this up, as it would affect chain rebuilds in the future - `autorebuild_canceled` should be set in this case. Fixed